src/store/token-swap/token-swap.thunks.js

```diff
@@ -59,4 +60,31 @@ function fetchAccounts (fromItem) {
   }
 }
 
-export { fetchAccounts }
+/**
+ * Get Quotes for a token swap between pairs
+ * @param {Object} data
+ * @param {String} data.fromToken - contract address from Token that user wants to Swap
+ * @param {String} data.toToken - contract addres from Token that user wants to Get
+ * @param {String} data.fromHezAddr - address with tokens to swap
+ * @param {String} [data.amountFromToken] - amount that user wants to swap
```

Comment:
I think that now the `[` and `]` are not needed here and below right?

Reply:
They have the same meaning that in property or maybe do you want to use = instead []?

Reply:
What do they mean? I think we can remove them too

Reply:
Brackets means that the param is optional, String= means the same.